src/site/filters/liquid.js

```diff
@@ -709,6 +709,24 @@ module.exports = function registerFilters() {
     return string.replace(regex, newVal);
   };
 
+  liquid.filters.processCentralizedContent = (entity, contentType) => {
+    // TODO - add more cases as new centralized content types are added
+    // eslint-disable-next-line sonarjs/no-small-switch
+    switch (contentType) {
+      case 'wysiwyg': {
+        return {
+          fieldWysiwyg: {
+            // eslint-disable-next-line camelcase
+            processed: entity?.field_wysiwyg[0]?.processed,
+          },
+        };
+      }
+      default: {
+        return entity;
+      }
+    }
+  };
```

Comment:
Small nit - I kinda prefer “processDynamicContent” or “processDynamicParagraphType” because using the word “dynamic” here helps tie it into the template.